study.md

```diff
@@ -27,17 +27,17 @@ it's also not a rendered page.
 Please name two things the DOM is not.
 
 ```md
-<!-- your answer here -->
+<!-- It is not what we see on the web (a rendered page) nor its source -->
 ```
 
 Is the information inside your Dev Tools the DOM? Why or why not?
 
 ```md
-<!-- your answer here -->
+<!-- no, the Dev tools are temporary and do not change the linked files. -->
```

Comment:
The DOM is an interface that represents HTML documents in a tree structure.  What you see in your web browser's developer tools is a visual representation of the browser's implementation of that interface.